README.md

```diff
@@ -454,17 +464,20 @@ with Stream(two_rows_source, sample_size=1) as stream:
 
 Data sample could be really useful if you want to implement some initial data checks without moving stream pointer as `stream.iter/read` do. But if you don't want any interactions with an actual source before first `stream.iter/read` call just disable data smapling with `sample_size=0`.
 
-### Allow html
-
-By default `Stream` will raise `exceptions.FormatError` on `stream.open()` call if html contents is detected. It's not a tabular format and for example providing link to csv file inside html (e.g. GitHub page) is a common mistake.
+### Bytes sample size
 
-But sometimes this default behaviour is not what is needed. For example you write custom parser which should support html contents. In this case `allow_html` option for `Stream` could be used:
+On initial reading stage `tabulator` should detect contents encoding. An argument `bytes_sample_size` allow to customize how much bytes will be read to detect encoding:
```

Comment:
Instead of, "An argument `bytes_sample_size` allow to customize how much bytes will be read to detect encoding", I suggest:

"The argument `bytes_sample_size` customizes how many bytes will be read to detect encoding"

Reply:
Fixed. Thanks!